### ISSUE TITLE
UnicodeEncodeError except when running portray

### ISSUE THREAD

**Issue by shazz** [timothycrosley/portray#2]
Hi,

I just tried portray but unfortunately, I cannot even start it.
I'm using:

````
$ python --version
Python 3.7.0
$ pip freeze | grep portray
portray==1.0.4
````

````
$ portray
Traceback (most recent call last):
  File "c:\python\python37\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python\python37\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\git\test\.venv\Scripts\portray.exe\__main__.py", line 9, in <module>
  File "c:\git\test\.venv\lib\site-packages\hug\api.py", line 437, in __call__
    print(str(self))
  File "c:\git\test\.venv\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters in position 4-10: character maps to <undefined>
````

Any idea ?

Looks to be an UTF-8 problem, maybe because I'm on Windows/Git Bash and your nice ASCII logo ?
https://stackoverflow.com/questions/14630288/unicodeencodeerror-charmap-codec-cant-encode-character-maps-to-undefined

Thanks!

**Comment by defnull** [comment]
Happens also on servers or within docker containers with `LANG=C`.

**Comment by shazz** [comment]
Sorry for your nice logo ^^

**Comment by timothycrosley** [comment]
Worth it so more people can enjoy portray! Thanks for letting me know you encountered this issue. It should now be fixed in release 1.1.0: https://timothycrosley.github.io/portray/CHANGELOG/#changelog
